src/components/activity-groups-list/activity-group-item/activity-item/interaction.tsx

```diff
@@ -33,6 +39,9 @@ export const Interaction: FC<ActivityItemProps> = memo(({ activity }) => {
     case ActivitySubtype.ChangeAllowance:
       return <ChangeAllowance activity={activity as AllowanceInteractionActivity} />;
 
+    case ActivitySubtype.Route3:
+      return <Route3 activity={activity as Route3Activity} />;
```

Comment:
can we avoid assertion?

Reply:
Yes, we need to make a refactor with typeguards (the same problem is on the extension), but i think it would be better to do it after all activity types will be added